index.html

```diff
@@ -9248,18 +9248,20 @@ <h3 id="example-2">
             <code>reduction</code> property on the
             <a>DynamicsCompressorNode</a>
             </li>
-            <li>The compression curve is has three parts:
+            <li>The compression curve has three parts:
               <ul>
-                <li>The first part is the identify: \(f(x) = x\).
+                <li>The first part is the identity: \(f(x) = x\).
                 </li>
                 <li>The second part is the soft-knee portion, which MUST be a
                 monotonically increasing function.
                 </li>
                 <li>The third part is a linear function of equation \(f(x) =
```

Comment:
Nit:  "linear function of equation" -> "the linear function:"

The ":" to make to match what you say in line 9253 above.